.gitignore

```diff
@@ -23,4 +23,19 @@ target/
 EvaluationData
 *.DS_Store
 *.polyglot
-*.pom.tycho
\ No newline at end of file
+*.pom.tycho
```

Comment:
We could remove this file type I suppose?

Reply:
good catch :+1: